packages/bpk-component-section-list/src/bpk-section-list-item.scss

```diff
@@ -0,0 +1,52 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-section-list-item {
+  $root: &;
+
+  display: flex;
+  width: 100%;
+  min-height: $bpk-spacing-lg * 2;
+  padding: $bpk-spacing-xs $bpk-spacing-sm;
+  justify-content: space-between;
+  align-items: center;
+  border: none;
+  background-color: $bpk-color-white;
+  color: $bpk-color-gray-700;
+  text-decoration: none;
+
+  @include bpk-border-bottom-sm($bpk-color-gray-100);
+
+  &__chevron {
+    fill: $bpk-color-gray-500;
+
+    @include bpk-rtl {
+      transform: scaleX(-1);
```

Comment:
We should handle this in JS instead of CSS by using the approriate icons imo

Reply:
Yeah that's fair, I'll do that.

Reply:
Actually I'm not sure how to do this in JS, we don't generally access the RTL state in there do we? Do you know of any code samples I can check out?

Reply:
Navigation stack is doing this. see: https://github.com/Skyscanner/backpack/blob/dda0a9e869bf3e3936b32efc1d7ca0a5f21b4b4c/packages/bpk-component-navigation-stack/stories.js#L35. There's an internal `isRTL` function here https://github.com/Skyscanner/backpack/blob/dda0a9e869bf3e3936b32efc1d7ca0a5f21b4b4c/packages/bpk-component-navigation-stack/src/utils.js

Reply:
Hmm we have `withRtlSupport` but that actually just adds a class anyway... Since you are using an icon could you at least change the code to use `withRtlSupport`?

Reply:
Forgot `withRtlSupport` was a thing! Thanks both of you, I've updated the code.